smf/sled-agent/config.toml

```diff
@@ -19,7 +19,9 @@ zpools = [
   "oxp_f4b4dc87-ab46-49fb-a4b4-d361ae214c03",
 ]
 
-# An optional data link on which VNICs should be created.
+# An optional data link from which we extract a MAC address.
```

Comment:
Note to self: this comment isn't *entirely* accurate, we still allocate the "sled address" on a physical link. TBD if we will still do that with this PR is closer to merging.

Reply:
Nope! It ended up staying accurate, since we moved GZ address allocation to the etherstub vnic.